src/core/qgsfeature.h

```diff
@@ -598,6 +598,8 @@ class CORE_EXPORT QgsFeature
       sipRes = sipConvertFromNewType( v, sipType_QVariant, Py_None );
     }
     % End
+#else
+    QVariant attribute( const QString &name ) const;
```

Comment:
Is there any chance this could be reversed (i.e. sip code after c++ header code)? I think it would feel more natural to first read the "real" declaration and then the sip-approach.